luajava/src/main/java/party/iroiro/luajava/Lua.java

```diff
@@ -759,6 +759,19 @@ public interface Lua extends AutoCloseable {
      */
     void createTable(int nArr, int nRec);
 
+    /**
+     * Creates a new empty table and pushes it onto the stack
+     * 
+     * Does the equivalent to createTable(0, 0)
+     * 
+     * <p>
+     * The new table but without pre-allocation.
```

Comment:
Please base the Javadoc on the Lua documentation: https://www.lua.org/manual/5.1/manual.html#lua_newtable .

Reply:
Also, make sure to use `{@link ...}` to link to `createTable`.